cmd/crossbuild.go

```diff
@@ -51,6 +51,9 @@ var (
 		"openbsd/arm": {"openbsd/armv7"},
 		"netbsd/arm":  {"netbsd/armv6", "netbsd/armv7"},
 	}
+	defaultS390XPlatforms = []string{
```

Comment:
Do we want to call this `defaultS390Platforms`, just in case we ever add support for both `s390` and `s390x`? Or is that silly?

Reply:
No, it should be `defaultS390Platforms`. It also needs to be moved up to below `defaultMIPSPlatforms`.